grpcclient/lc_client.go

```diff
@@ -166,14 +167,20 @@ func NewLcClient(setters ...LcClientOption) *LcClient {
 		setter(cli)
 	}
 
+	retryOpts := []grpc_retry.CallOption{
+		grpc_retry.WithMax(3),
+		grpc_retry.WithBackoff(grpc_retry.BackoffExponential(500 * time.Millisecond)),
```

Comment:
I'd start with a smaller initial delay - like 100ms in case there's some really short glitch on the network, but use much higher number of retries.

Reply:
OK, let's then first see how it goes in practice.